test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ComplexTypeInheritance/ComplexTypeInheritanceTests.cs

```diff
@@ -48,6 +48,116 @@ public ComplexTypeInheritanceTests(WebHostTestFixture fixture)
             }
         }
 
+        public static TheoryDataSet<string, string, string,bool> NewCollectionMembers
+        {
+            get
+            {
+                return new TheoryDataSet<string, string, string,bool>
+               {
+
+                   { "convention", @"
+{
+        '@odata.type':'#Microsoft.Test.E2E.AspNet.OData.ComplexTypeInheritance.Polygon',
+        'HasBorder':true,'Vertexes':[
+            {'X':21,'Y':12},
+            {'X':32,'Y':23},
+            {'X':14,'Y':41}
+        ]
+}",
```

Comment:
you can combine the tests using codes. See "MediaTypes" foreach (...) clause.

Reply:
Done.